### PR TITLE
fix(general-cs-quiz.ts) : added new question

### PR DESCRIPTION
**Pull request by Chirag8023**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -734,6 +734,15 @@ const generalCSQuiz = [
     Explanation: "Lossless compression is a compression algorithm that allows the original data to be perfectly reconstructed from the compressed data.",
     Link: "https://www.freecodecamp.org/news/image-file-types-picture-format-extensions-jpeg-gif-png-svg-tiff/",
   },
+  {
+    Question: "What is a Nibble?",
+    Answer: "Nibble is a four-bit aggregation",
+    Distractor1: "Nibble is an algorithm",
+    Distractor2: "Nibble is a octet",
+    Distractor3: "Nibble is a device",
+    Explanation: "a nibble is group of four bits or it's half of a byte.",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:743]:
```suggestion
    Explanation: "A Nibble is a group of four bits (half of a byte).",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Chirag8023** [comment]
Done changes
👍🏻

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**